dart/lib/src/hub.dart

```diff
@@ -309,6 +314,115 @@ class Hub {
       }
     }
   }
+
+  // maybe add optional params for sampled and customSamplingContext
+  ISentrySpan startTransaction(
```

Comment:
maybe add optional params for sampled and customSamplingContext
right now only `startTransactionWithContext` can use the customSamplingContext databag

Reply:
`customSamplingContext` can be added to `startTransaction(name, op)` and question the boolean passed to startTransaction

Reply:
startTransaction with explicit boolean: https://develop.sentry.dev/sdk/performance/#sampling

Reply:
lets keep it for later, we can always add it later